### ISSUE TITLE
Feature request: export/copy REPL history

### ISSUE THREAD

**Issue by ricardojmendez** [cursiveclojure/cursive#876]
After running a series of experiments on the REPL, it would be really useful to be able to export to a file or copy to the clipboard the entire history of REPL commands.

**Comment by cursiveclojure** [comment]
Good idea - this should be pretty easy. I'm assuming you'd want the commands, but not the results? I have a long term plan to semi-automatically make unit tests out of a REPL session, but that's harder than just exporting.

**Comment by ricardojmendez** [comment]
Yes, just the commands would be fine.   It would save a lot of set up on repeated manual experiments, like requires and initial defs.

**Comment by CRHough** [comment]
Any progress on this?

**Comment by Alcamech** [comment]
Any progress on this ? Recording REPL history to a file automatically during each session would be nice.

**Comment by billvanyo-droit** [comment]
Until it's a feature, I'm doing this to print a project's repl history:
```clj
(require '[sigel.xpath.core])
(defn print-cursive-repl-history []
  (let [workspace-xml (slurp ".idea/workspace.xml")
        selector "//*[name()='component' and @name='ReplState']/node()"
        repl-map (clojure.edn/read-string (str (sigel.xpath.core/select workspace-xml selector)))
        cmds (mapv #(get % :command) (:local (:repl-history repl-map)))]
    (mapv println cmds)
    nil))
```

**Comment by tamasjung** [comment]
Or 
`xmllint --noent --xpath "//*[name()='component' and @name='ReplState']/node()" workspace.xml | jet -q '[:repl-history :local (map :command) ]'  | bb -e "(run!  println *input*)"`
